### PR TITLE
Randy - challenge 1

### PR DESCRIPTION
**Pull request by io-world**

### CODE REVIEW

**Comment by io-world** [comment]
Add Version to Color one

File: background-colors.suite/Scripts/randy2.script [hunk 1]
```diff
@@ -0,0 +1,54 @@
+﻿//You know like an actual comment that tell you what the script is about
```
Comment (CosmicKetchup) [inline background-colors.suite/Scripts/randy2.script]: @io-world Randy, just make sure you include the connection process within your script(s). These are meant to be self-contained solutions.

File: background-colors.suite/Scripts/randy2.script [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+put keys(bgColor) into bgKeys
+repeat bgCount times
+	
+	put item repeatindex() of bgKeys into SingleBGKey
+	if imagefound(text:SingleBGKey) then
+		put FoundImageInfo().ImageRectangle into FiiR
+		put bgColor.(SingleBGKey) into TColor
+		put me.DefineColorArea (FiiR) into (CountColorMatrix,ColorMatrix)
+		if CountColorMatrix.(TColor) >=1
+			LogSuccess	"Found:"&& SingleBGKey && "with Pixel count"&&CountColorMatrix.(TColor) &&"Color" && TColor && "Within Area" && FiiR
```
Comment (CosmicKetchup) [inline background-colors.suite/Scripts/randy2.script]: I'm not seeing any validation that the table values are listed in the proper vertical order. Let me know if I missed something.

**Review by CosmicKetchup — CHANGES_REQUESTED** [review]
These solutions seem very similar. Could you condense them into a single solution?

**Review by CosmicKetchup — APPROVED** [review]

**Pull request merged**